manifests/g/gaphor/gaphor/2.24.0/gaphor.gaphor.locale.en-US.yaml

```
@@ -0,0 +1,65 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: gaphor.gaphor
+PackageVersion: 2.24.0
+PackageLocale: en-US
+Publisher: Gaphor Community
+PublisherUrl: https://gaphor.org/
+PublisherSupportUrl: https://gaphor.org/discuss
+Author: gaphor
+PackageName: Gaphor
+PackageUrl: https://github.com/gaphor/gaphor
+License: Apache-2.0 license
+Copyright: Copyright (c) 2024 Gaphor Contributors
+CopyrightUrl: https://github.com/gaphor/gaphor/blob/main/LICENSE.txt

```

```suggestion
# CopyrightUrl: 
```

---

Maybe, link to the new directory instead?

```suggestion
CopyrightUrl: https://github.com/gaphor/gaphor/tree/main/LICENSES

```